src/core/ghosts.cpp

```diff
@@ -36,6 +36,7 @@
 #include "communication.hpp"
 #include "grid.hpp"
 #include "particle_data.hpp"
+#include "domain_decomposition.hpp"
```

Comment:
Why is this needed?

Reply:
Because of
https://github.com/bindgens1/espresso/blob/c20273e50ab7b3707a6e9aab93250c2c2478e5d2/src/core/ghosts.cpp#L340
https://github.com/bindgens1/espresso/blob/c20273e50ab7b3707a6e9aab93250c2c2478e5d2/src/core/ghosts.cpp#L343
https://github.com/bindgens1/espresso/blob/c20273e50ab7b3707a6e9aab93250c2c2478e5d2/src/core/ghosts.cpp#L469
https://github.com/bindgens1/espresso/blob/c20273e50ab7b3707a6e9aab93250c2c2478e5d2/src/core/ghosts.cpp#L472